### PR TITLE
Ability to add repositories via a link

### PR DESCRIPTION
**Pull request by ConradWeiser**

### CODE REVIEW

**Comment by ConradWeiser** [comment]
Hold off on merging this, the URLs need to be changed to include the https:// - But the special characters need to be encoded properly

**Comment by FramboisePi** [comment]
Urls have been updated to work with the app.

---

The format used is
```
paperback://addRepo?displayName=Beta%20Extensions&url=https://conradweiser.github.io/extensions)
```
They need the `https` part, encoded specials characters (`%20` for spaces) and no quote.

**Comment by ConradWeiser** [comment]
Updated, and is ready to be merged. Regarding the merge conflict, please allow this branch to overwrite anything in master. Things should fall into place properly.

**Review by FaizanDurrani — APPROVED** [review]

File: src/help/guides/adding-repos.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ lang: en-US
...
-| **H-Extensions**      | https://paperback-ios.github.io/h-extensions | <ExtensionsList url="https://paperback-ios.github.io/h-extensions"/> | Official repository for 18+ sources. These also include redirector versions of the source, which allow you to view content even if it is blocked in your country, without a VPN |
-| **MangaDex Unlocked** | https://pogogo007.github.io/extensions-beta | <ExtensionsList url="https://pogogo007.github.io/extensions-beta"/> | This source overwrites MangaDex and unlocks the full contents of the website. No content is filtered. May load faster than the built-in version of the source |
-| **Beta Extensions**   | https://conradweiser.github.io/extensions | <ExtensionsList url="https://conradweiser.github.io/extensions"/> | These extensions are upcoming to the official repository. These sources may or may not work |
+| Repository Name | Base URL | Available Sources | Notes | Add Repo (iOS Only) |
+| :-------------: | :------: | :---------------- | :---: | :---:
```
Comment (FramboisePi) [inline src/help/guides/adding-repos.md:24]: Hi @ConradWeiser,
Was there a particular reason to left align the column title or is this an unintended change? 
In theory the extension list should be left aligned by itself.

**Pull request merged**